Sources/Kommunicate/Classes/Extensions/UIViewController+Extension.swift

```diff
@@ -62,7 +62,7 @@ extension UIViewController {
     }
 
     class func topViewController() -> UIViewController? {
-        return topViewControllerWithRootViewController(rootViewController: UIApplication.shared.keyWindow?.rootViewController)
+        return topViewControllerWithRootViewController(rootViewController: UIApplication.main?.keyWindow?.rootViewController)
```

Comment:
We'll be returning nil as value if shared is not available. Better to add a guard statement before this and return from there.

Reply:
done